### DISCUSSION TITLE
如何用post /user  创建用户？

### DISCUSSION THREAD

**Discussion by rain2307** [usememos/memos#1838] [Q&A] [answered]
有没有例子啊，go代码 有点不熟，谢谢

https://github.com/usememos/memos/blob/f1d85eeaec5c2e00e0a3181a31861ddef76b91fd/server/user.go#L19

**Comment by Zeng1998** [comment, marked as answer]
https://github.com/usememos/memos/blob/main/web/src/helpers/api.ts
